gauntlet/packages/gauntlet-solana-contracts/src/commands/contracts/ocr2/setBilling.ts

```diff
@@ -44,6 +46,15 @@ export default class SetBilling extends SolanaCommand {
     const input = this.makeInput(this.flags.input)
 
     const info = await program.account.state.fetch(state)
+    const tokenVault = new PublicKey(info.config.tokenVault)
+    const [vaultAuthority] = await PublicKey.findProgramAddress(
+      [Buffer.from(utils.bytes.utf8.encode('vault')), state.toBuffer()],
+      program.programId,
+    )
+    const payees = info.oracles.xs
+      .slice(0, info.oracles.len)
+      .map((oracle) => ({ pubkey: oracle.payee, isWritable: true, isSigner: false }))
```

Comment:
thinking about if this works on a initialization edge case where oracles are not configured yet.
for example: the flow command follows set_billing => propose config
(i will test this out locally)

Reply:
verified this locally that `set_billing` still works when a config has not been set

Reply:
Yep, in that case `oracles.len() == 0` so the contract also expects 0 `remaining_accounts`